loginPrcess.php

```diff
@@ -26,6 +26,11 @@
 
         if ($loginNumRows2 == 1) {
             $loginData2 = $loginResultSet2->fetch_assoc();
+
+            $user = new stdClass();
+            $user->id = $loginData2["id"];
```

Comment:
You can assign the `user` object to the `user session` associative array. Its very useful when you have to access user related data. And you can reduce additional search processes.